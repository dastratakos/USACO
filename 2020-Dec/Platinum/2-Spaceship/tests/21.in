60 60 60
111110000010000110111011100010111101111110010110001001110101
011110011101101001111100011100110110011010011110110100111101
110110110111010010101110110011001100111101111000000011100111
101111111011011110100111111001101001101011011111111110100111
010110100101111110101111101111110010111100011110101010110111
101110011001110110110111111001011111101011111100001111011110
010011111011100000110111011110101010011101001111111011111100
111011110110011010100110101011011111111110111111011101101111
111111101111001110000010110011111010011101011101110111010100
001110011111101111101001110110001000101110101011111111111110
111110111101011011111111111110001111111000111110101101111111
111101010001111111100010110000111111101000011011111111111110
101111011100101001111111110101110111110001111101010000011110
001011010100111111110011111111111110001010011111011011111011
110011001011111000010110011011111111111101101111100010111111
101110111111111110110001111111001111111111001011111111111010
101010011010111111100111011010101110111111111111010110100110
000111001110111100011111001100101111101011111101101111101011
111011111110111101110100101101111001110011000111011011101100
110111101010111111101011111111101011111100101110101101001000
110111110111100110111101111011110111001001111101010110001101
001010111111110110010001111111011110011011111111101101010101
010111110110101111100110110011111010100111111010110111001101
001011100110010111111111101101110111111101001111001111101111
011111111111111111010011111101110101110111011101111110101101
100101000110111011111010110001001100110110101111011110110110
010111110111101001011100111101101100011011001111111101011101
111110110110111110111011111010101000010110011111101011101111
111110111110101101010100101111111011011111000010011001111010
011010011001011111111001110101111111000011111011111011111011
111110011110111110010111111111111010001111110000111111111000
010101111111011111110110110011011000110111100011001111111000
100011110011101111110011001111110100110110011110110111001001
001110001011111111100000101001011101011101000011100101111101
111111001011100010101100010100111111100011111011111111111001
010000111111011101001110011011011111100011011110111100111111
111111001000101111100111101011110001011000011101110111101110
010000101111010111101111111111111001011111010111011011010110
111101111010111111101111110110011111100101110110011011101101
101110011011000111001010111110110101001111011010110111111001
111100010111001110011110110101000111111011101110010110101101
111011001011000011011110010110110011111011101001101111010100
011110110101111110110111111110000111110101011001011011010111
011110010010111111110101100110111001001011110110101111111111
010101111110011111011001011001111111100110101011011011101111
101011100100001011010001111111011011101011000111111011011111
010111111110001011111111011111000111101111110001101111110111
110010100001111111011111001011100101001000100110001011110000
110110100011111001101111000001111101101101111010100011111111
111101110110100011011110110110111100101101110111110001100011
110101111011011010011110110100111101111111111101001110100001
011011001111101101001110010111110011111101101011110010110111
010010110111010111110111101010011111100111111011111110111101
000011101110101101111110010011111111010101110000111011101011
111111010110010111010111111111011101001101110111010011111011
001111111111101110111011111111011111011011101110100100101010
111011000011101111110011011101001111111100001110110101110110
111101111100111001111010100001111011110111101000111101111111
011110101101110011010111010111110110010111001111111011111111
110101010101001111101011010100101000111101101111111100111001
2 26 5 51
2 44 3 31
4 36 2 36
59 4 54 2
5 14 5 42
1 26 50 55
3 21 45 8
5 38 1 2
1 12 14 18
2 23 2 13
1 33 28 3
2 12 1 23
1 53 2 12
4 46 4 58
58 34 3 42
15 30 5 60
1 50 2 10
3 36 24 38
2 14 4 7
1 12 4 18
4 52 4 25
5 19 4 11
1 23 3 21
4 59 1 56
45 9 26 50
2 45 3 45
2 39 1 24
1 38 5 20
4 60 4 31
3 23 4 32
5 33 19 37
1 36 51 14
1 47 25 39
5 33 3 20
3 50 41 16
2 53 4 36
3 11 28 15
29 19 3 14
3 12 2 15
4 13 2 1
4 39 5 40
24 5 2 14
45 47 5 56
1 59 4 28
3 35 13 50
3 35 5 56
1 16 55 50
43 44 4 31
2 43 29 30
1 31 1 32
13 28 4 14
2 11 24 15
37 57 2 19
12 28 1 52
4 13 2 33
5 51 1 56
5 24 1 10
4 11 1 36
5 32 4 57
1 33 1 33
