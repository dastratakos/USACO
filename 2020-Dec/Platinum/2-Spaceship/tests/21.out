630035150
682026759
927448582
344175720
991917202
846723358
155627817
417466638
819194716
662083409
165032361
328378651
311098462
193167835
342416826
621137325
912696242
609886239
626403642
320465271
497344612
837896624
114588146
604245638
834034487
514883892
192510605
213723624
793489356
475474682
320830908
361248822
214312307
363568165
936009945
738822415
650002055
502882245
153419620
497358990
85623645
327775863
651300028
704040041
529758287
933388369
158709561
768827612
197559519
784109738
699795637
991328249
447319163
67772944
940228828
846891992
883486796
197574204
850342017
883047422
